spark/processing/3.1/setup.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This file seems duplicated with py39, and py37 is somehow a binary file not visible. 

Are py37 and py39 generated by build? Should we exclude them?

Reply:
removed